.mci.yml

```diff
@@ -840,8 +840,6 @@ functions:
             set -o errexit
             set -o pipefail
             set -x
-            echo "Building Alpine Docker image"
-            make -C extras/docker/alpine3.18 nocachebuild test
             echo "Building Debian Docker image"
```

Comment:
Suggest commenting out the alpine build with a comment referencing MONGOCRYPT-601:

```yml
            # TODO: Re-enable Alpine test once MONGOCRYPT-601 is released.
            # echo "Building Alpine Docker image"
            # make -C extras/docker/alpine3.18 nocachebuild test
```